community/rfcs/24-08-02-OPEA-AIAvatarChatbot.md

```diff
@@ -0,0 +1,140 @@
+## Title
+<!-- The short description of the feature you want to contribute -->
+Using OPEA to implement a RAG-Powered Human-Like AI Avatar Audio Chatbot (WIP)
+
+## Author
+<!-- List all contributors of this RFC. -->
+[ctao456](https://github.com/ctao456), [alexsin368](https://github.com/alexsin368), [YuningQiu](https://github.com/YuningQiu), [louie-tsai](https://github.com/louie-tsai)
+
+## Status
+<!-- Change the PR status to Under Review | Rejected | Accepted. -->
+v0.1 - ASMO Team sharing on Fri 6/28/2024  
+[GenAIComps pr #400](https://github.com/opea-project/GenAIComps/pull/400) (Under Review)
+[GenAIExamples pr #523](https://github.com/opea-project/GenAIExamples/pull/523) (Under Review)
+
+## Objective
+<!-- List what problem will this solve? What are the goals and non-goals of this RFC? -->
+* Introducing a new microservice, animation, that generates animated avatars from audio and image/video inputs; and a new megaservice, AvatarChatbot, that integrates the animation microservice with the existing AudioQnA service to build a human-like AI audio chatbot.
```

Comment:
possible to define how important to have AI avatar and what impact we might have if we don't have it? try to take some information from our competitor: https://nvidianews.nvidia.com/news/digital-humans-ace-generative-ai-microservices

Reply:
Revised accordingly